Automattic/vip-design-system
# PIE-2949 Adds basic a11y assertions with @axe-core/react

## Description

Adds a11y checks powered by @axe-core/react

![Screen Shot 2021-12-02 at 14 55 51](https://user-images.githubusercontent.com/156388/144477269-c91f89ee-9224-4809-816d-e8dcea0937a1.png)

Issues will aso be reported on the console when browsing storybook:

![Screen Shot 2021-12-02 at 15 11 32](https://user-images.githubusercontent.com/156388/144479564-cdb56794-fb2a-4005-ad75-4db32eb3ffd7.png)

## Checklist


## Steps to Test

1. Pull down PR.
2. Remove the `alt` attribute from the avatar component
3. `npm test`
4. Verify that accessibility tests fail
